### ISSUE TITLE
Source build fails after Git repo change

### ISSUE THREAD

**Issue by derchrisuk** [mika/jenkins-debian-glue#199]
I have a running source/binary from my personal git repo.
It build successfully, and produced a .deb file and added it to our apt repo.
Now we have moved it to a group git.
After I changed the Git URL for the source, it fails to build:

`dpkg-genchanges: error: cannot fstat file ../timelens_0.1.0_amd64.buildinfo: No such file or directory`

Both my and the new Git repo are identical.
And this file is being created by the build server.

Anyway to fix this?

**Comment by mika** [comment]
I'd need to full build log to see what's going on, feels to me like there was a previous error already?
Are you sure the git clone/fetch actually works? Try removing all the workspace from the source job and rerun.

**Comment by mika** [comment]
So `dpkg-buildpackage` fails, because `dpkg-genchanges` can't find the file it expected. The version information from `timelens_0.1.0_amd64.buildinfo` completely disagrees with the one from `timelens_0.1.0+0~20181015150809.44~1.gbpbabc9f.*`.

1) Does a file ../timelens*buildinfo exist?
2) Is there a file debian/files present? If so, what's inside it? Is that file part of the git repository? Run `fakeroot debian/rules clean` and show `git status` output.
3) The `debian` branch contains what you expect it to build?

I suppose manual execution of the following commands on the git repository also fails for you?

```
git branch debian remotes/origin/debian
git checkout -b jenkins-debian-glue-buildbranch21572 babc9fbb67e1b044248ce8eced8019499fabdcd6
gbp dch --auto -S --multimaint-merge --ignore-branch --new-version=0.1.0+0~20181015150809.44
gbp buildpackage -nc --git-force-create --git-ignore-new --git-ignore-branch -S -us -uc --git-verbose --git-builder=/bin/true --git-cleaner=/bin/true --git-export-dir=
dpkg-buildpackage -uc -us -nc -d -S
```

**Comment by derchrisuk** [comment]
I ran the few commands on both the repos, and yes it was failing also manually.
But only the repo which I changed to, the original repo was working fine
So I checked again both repos.
It turns out there is a slight difference in one of the files, which makes the build fail.
Shouldn't the command fail with some comments from the make process?
Anyway, I know where the error is coming from.
What I don't see at the moment how to fix it.
Because the new repos HEAD does have the fixed included, but still fails to build
